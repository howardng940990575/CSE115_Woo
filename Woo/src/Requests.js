import React, { Component } from 'react';
import { StyleSheet, Text, View, Button, TextInput, Image, Animated, TouchableOpacity, Dimensions, TouchableHighlight, YellowBox } from 'react-native';
import * as firebase from "firebase";

export default class Requests extends Component {

    onPressRequests = () => {

        this.props.navigation.navigate('PatientHomepage')

    }

    render() {
        return (
            <View style={styles.container}>
             <Text>Requests Screen Homepage</Text>
             <Button 
                title = 'Go back'
                onPress={this.onPressRequests} />
         </View>
        );
    }
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: '#72C3C9',
        alignItems: 'center',
        justifyContent: 'center',
    },
});