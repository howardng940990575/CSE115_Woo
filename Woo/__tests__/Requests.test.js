import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Requests from '../src/Requests';

test('Request SnapShot', ()=> {
    const snap = renderer.create(<Requests />).toJSON();
    expect(snap).toMatchSnapshot();
});
