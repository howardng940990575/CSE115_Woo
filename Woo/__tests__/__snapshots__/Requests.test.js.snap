// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Request SnapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#72C3C9",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Requests Screen Homepage
  </Text>
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Go back
      </Text>
    </View>
  </View>
</View>
`;
